### PR TITLE
Warn about refs in add_event_handler

### PR DESCRIPTION
**Pull request by almarklein**
This is a proposal for #318.

### CODE REVIEW

File: wgpu/gui/base.py [hunk 1]
```diff
@@ -301,17 +301,19 @@ def handle_event(self, event):
                 callback(event)
 
     def add_event_handler(self, *args):
-        """Register an event handler.
+        """Register an event handler to receice events.
```
Comment (Korijn) [inline wgpu/gui/base.py:304]: Typo :)

**Review by Korijn — APPROVED** [review]

File: wgpu/gui/base.py (hunk 1 above)
Comment (almarklein) [inline wgpu/gui/base.py:304]:
```suggestion
        """Register an event handler to receive events.
```

Comment (almarklein) [reply wgpu/gui/base.py]: I find the idea of running a spellchecker on CI increasingly interesting 🤔

Comment (Korijn) [reply wgpu/gui/base.py]: Yeah me too 😅

**Pull request merged**